emukit/bayesian_optimization/acquisitions/expected_improvement_per_cost.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple, Union
```

Comment:
I have a proposal for a different implementation of cost. Instead of creating <SomeAcquisitionFunction>PerCost, why don't we make use of acquisitions algebra? Let's define a general CostAcquisition. Then all we need to do in CostSensitiveBOLoop would be just divide, and that's it.

The worry here is that we may end up duplicating a lot of stuff, having cost-aware and cost-unaware versions of our code. So I am trying to think of a good way to keep this duplication at a minimum.

It also has an advantage that this cost acquisition function can be re-used in other packages, such as quadrature or exp design.

Makes sense?